README.md

```diff
@@ -58,7 +65,7 @@ docker run --name test-mysql -p 3306:3306 -e MYSQL_ROOT_PASSWORD=<set_password>
 Create a `config` folder at the root of the solution
 
 ```bash
-mkdif config
+mkdir config
 ```
 
 Add a new file `logback.xml` in the config
```

Comment:
Is there a reason why there are instructions for creating this file and it's associated directory in the readme instead of just having the file and directory directly in the repo?

Reply:
no real reason, I think it should be in the repo has well I will create a backlog for this